package.json

```diff
@@ -145,6 +145,7 @@
     "react-dev-utils": "6.0.0-next.66cc7a90",
     "react-docgen": "2.21.0",
     "react-dom": "^16.8.5",
+    "react-hot-loader": "4.8.4",
```

Comment:
This is to fix https://github.com/pedronauck/docz/issues/793
Not ideal, but hopefully temporary.